.rubocop.yml

```diff
@@ -131,4 +131,5 @@ Lint/SelfAssignment:
   Enabled: false
 Lint/RedundantCopDisableDirective:
   Enabled: false
-
+Lint/MissingSuper:
+  Enabled: false
```

Comment:
can you explain what this is? i never really got `super` but disabling a linter makes me nervous :)

Reply:
Yeah. I think will remove this. I looked around at other codebases using ViewComponent and probably the solution is to call `super` in the method instead of disabling the cop.

Reply:
ohhh i see so otherwise it would monkeypatch it? makes sense :)

Reply:
Resolved in [9b10173](https://github.com/geeksforsocialchange/PlaceCal/pull/2638/commits/9b1017332dcc523fbb76326bca1dec6ad9ba321f)